File: kyonggiyo-core/src/main/java/kyonggiyo/domain/candidate/util/StatusConverter.java
package kyonggiyo.domain.candidate.util;

import kyonggiyo.domain.candidate.Status;
import org.springframework.core.convert.converter.Converter;

public class StatusConverter implements Converter<String, Status> {

    @Override
    public Status convert(String status) {
        return Status.from(status);
    }

}

File: kyonggiyo-core/src/main/java/kyonggiyo/application/candidate/domain/util/StatusConverter.kt
package kyonggiyo.application.candidate.domain.util

import kyonggiyo.application.candidate.domain.vo.Status
import org.springframework.core.convert.converter.Converter

class StatusConverter : Converter<String, Status> {

    override fun convert(status: String): Status {
        return Status.from(status)
    }

}
